I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-09.txt

```diff
@@ -0,0 +1,3752 @@
+   OTSi properties needed for optical impairment-aware path computation
+   including the spectrum occupied by each OTSi signal.  The model also
+   defines a pointer (leafref) from the OTSi to the transceiver module
+   terminating the OTSi signal.
+
+2.3.2.  Optical Tributary Signal Group (OTSiG)
+
+   The definition of the OTSiG is currently being moved from ITU-T
+   Recommendation G.709 [G.709] to the new draft Recommendation G.807
+   (still work in progress) [G.807].  The OTSiG is an electrical signal
+   that is carried by one or more OTSi's.  The relationship between the
+   OTSiG and the the OTSi's is described in ITU-T draft Recommendation
+   G.807, section 10.2 [G.807].  The YANG model below supports both
+   cases: the single OTSi case where the OTSiG contains a single OTSi
+   (see ITU-T draft Recommendation G.807, Figure 10-2) and the multiple
+   OTSi case where the OTSiG consists of more than one OTSi (see ITU-T
+   draft Recommendation G.807, Figure 10-3).  From a layer 0 topology
+   YANG model perspective, the OTSiG is a logical construct that
+   associates the OTSi's, which belong to the same OTSiG.  The typical
+   application of an OTSiG consisting of more than one OTSi is inverse
+   multiplexing.  Constraints exist for the OTSi's belonging to the same
+   OTSiG such as: (i) all OTSi's must be co-routed over the same optical
+   fibers and nodes and (ii) the differential delay between the
+   different OTSi's may not exceed a certain limit.  Example: a 400Gbps
+   client signal may be carried by 4 OTSi's where each OTSi carries
+   100Gbps of client traffic.
+
+
+
+
+
+
+
+
+
+
+Beller, et al.          Expires September 8, 2022               [Page 9]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model       March 2022
+
+
+                                  OTSiG
+           _________________________/\__________________________
+          /                                                     \
+                                    m=7
+   - - - +---------------------------X---------------------------+ - - -
+   / / / |                                                       | / / /
+    / / /|      OTSi         OTSi         OTSi         OTSi      |/ / /
+   / / / |        ^            ^            ^            ^       | / / /
+    / / /|        |            |            |            |       |/ / /
+   / / / |        |            |            |            |       | / / /
+    / / /|        |            |            |            |       |/ / /
+    -4  -3  -2  -1   0   1   2   3   4   5   6   7   8   9  10  11  12
+   --+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---
+                                   n = 4
+                  K1           K2           K3           K4
+
+      Figure 3: MC Example containing all 4 OTSi signals of an OTSiG
+
+2.3.3.  Media Channel (MC)
+
+   The definition of the MC is currently being moved from ITU-T
```

Comment:
Following #69 
we need to add the text "ITU-T G.807 describes a "media channel" as "media association that represents both the topology (i.e., the path through the media) and the resource (i.e., frequency slotor effective frequency slot) that it occupies"
In the scope of this draft, we occasionally use the term “channel” to indicate the resource of an MC (i.e., frequency slot or effective frequency slot), without representing a topology."

Reply:
Text added according to #69:
   <xref target="G.807"/> defines a "media channel" as "A media
   association that represents both the topology (i.e., the path
   through the media) and the resource (i.e., frequency slot or
   effective frequency slot) that it occupies." In this document,
   the term "channel" is occasionally used to indicate the resource
   of an MC (i.e., frequency slot or effective frequency slot),
   without representing topology.